VoicePolicy.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Policies;
+
+class VoicePolicy
+{
+    use HandlesAuthorization;
+
+    public function vote(User $user, Question $question)
+    {
+        return $question->user_id == $user->id;
```

Comment:
I think the user cannot vote on his questions. So the policy should be with "!=", don't you think?